.github/workflows/ci.yml

```diff
@@ -13,10 +13,66 @@ on:
       - release/**
 
 jobs:
+# add get-diff job 
+  get-diff:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      contract: ${{ steps.get-diff-contract.outputs.diff }}
+      certora-contract: ${{ steps.get-diff-certora.outputs.diff }}
+      scripts: ${{ steps.get-diff-scripts.outputs.diff }}
+      agents: ${{ steps.get-diff-agents.outputs.diff }}
+      env: ${{ steps.get-diff-env.outputs.diff }}
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get diff for contract
+        uses: technote-space/get-diff-action@v4.0.1
+        id: get-diff-contract # via outputs
+        with:
+          PATTERNS: |
+            +(contracts)/*.sol
+            +(contracts/**/*.sol
+
+      - name: Get diff for certora
+        uses: technote-space/get-diff-action@v4.0.1
+        id: get-diff-certora # via outputs
+        with:
+          PATTERNS: |
+            +(certora)/**/*.+(sol|spec|sh)
```

Comment:
at the moment, this would trigger the linter, even if there was only a change in the spec file, while it shouldn't.
I think, better to keep the spec file diff step separate from this one and use it as needed. Anyways, this step is called `certora-contract`, get's confusing when you have the spec as well within it. It should only be having contract diff. Can create a `certora-spec` diff step separately.

Reply:
Done